lax/lichens/sciencerun1.py

```diff
@@ -0,0 +1,187 @@
+"""The cuts for science run 0
+
+This includes all current definitions of the cuts for the first science run
+"""
+
+# -*- coding: utf-8 -*-
+import inspect
+import os
+from pax import configuration
+
+PAX_CONFIG = configuration.load_configuration('XENON1T')
+from lax.lichen import ManyLichen, StringLichen
+from lax.lichens import sciencerun0
+from lax import __version__ as lax_version
+
+# Store the directory of our data files
+DATA_DIR = os.path.join(os.path.dirname(os.path.abspath(inspect.getfile(inspect.currentframe()))),
+                        '..', 'data')
+
+
+class AllEnergy(ManyLichen):
+    """Cuts applicable for low and high energy (gammas)
+
+    This is a subset mostly of the low energy cuts.
+    """
+    version = lax_version
+
+    def __init__(self):
+        self.lichen_list = [
+            FiducialCylinder1T(),
+            InteractionExists(),
+            S2Threshold(),
+            InteractionPeaksBiggest(),
+            S2AreaFractionTop(),
+            S2SingleScatter(),
+            DAQVeto(),
+            S1SingleScatter(),
+            S1AreaFractionTop(),
+            S2PatternLikelihood(),
+            S2Tails()
+        ]
+
+
+class LowEnergyRn220(AllEnergy):
+    """Select Rn220 events with cs1<200
+
+    This is the list that we use for the Rn220 data to calibrate ER in the
+    region of interest.
+
+    It doesn't contain the PreS2Junk cut
+    """
+
+    def __init__(self):
+        AllEnergy.__init__(self)
+        # Replaces Interaction exists
+        self.lichen_list[1] = S1LowEnergyRange()
+
+        # Use a simpler single scatter cut
+        self.lichen_list[5] = S2SingleScatterSimple()
+
+        self.lichen_list += [
+            S1PatternLikelihood(),
+            S2Width(),
+            S1MaxPMT(),
+            SingleElectronS2s()
```

Comment:
@feigaodm: is this still needed after https://github.com/XENON1T/pax/pull/576?

If not, I guess we should remove from SR0 lichen too once SR0 gets reprocessed?